conda_forge_tick/deploy.py

```diff
@@ -148,22 +155,17 @@ def _deploy_batch(*, files_to_add, batch, n_added, max_per_batch=200):
                     pass
 
                 print(">>>>>>>>>>>> git push try", flush=True)
-                with sensitive_env() as env:
```

Comment:
Why is this removed?

Reply:
Because it is totally sufficient to call `get_bot_token()` to access the bot token and we don't need the complexity of an extra level of indentation.